### ISSUE TITLE
birds-of-ladakh is failing

### ISSUE THREAD

**Issue by benoit74** [openzim/zim-requests#1031]
### Recipe URL

https://farm.openzim.org/recipes/birds-of-ladakh

### Last log lines

```true
NA
```

### How many times the recipe failed in a row?

Many

### How many ZIM have been produced before failure?

Many

### Which action did you undertake so far?

I have requested the recipe again to confirm failure

### What's next?

I don't know

### More details

Problem is that:
- upstream server seems very unstable
- many Youtube videos are missing because they are not embedded

Not sure there is much we can do.

**Comment by Popolechien** [comment]
I will contact the website owner and see if there's something she can do.

**Comment by kelson42** [comment]
Thw two last ones have beeb successful. Closing.
https://farm.openzim.org/recipes/birds-of-ladakh

**Comment by benoit74** [comment]
Reopening, file is not yet in prod + ZIM is just not working at all : https://dev.library.kiwix.org/viewer#birds-of-ladakh_en_all_2025-07/birdsofladakh.org/

**Comment by kelson42** [comment]
@benoit74 What is not working? To me it works as good (or as bad) as the original We suite!

**Comment by benoit74** [comment]
Indeed, you're right, didn't realized original web site is now so broken. Wasn't that broken few months ago.

That being said, given the very bad quality of website, are we really open to publish such a ZIM. I'm quite sure we will have complaints like mine 🤣 

I propose to rather delete recipe from Zimfarm and consider we will not publish the ZIM for the time being.

**Comment by kelson42** [comment]
@Popolechien You decide?

**Comment by Popolechien** [comment]
No news from our contact, we can shut down the recipe I believe.

**Comment by benoit74** [comment]
Recipe deleted.
